app/seeders/basic_data/basic_data_seeder.rb

```diff
@@ -26,12 +26,34 @@
 #
 # See doc/COPYRIGHT.rdoc for more details.
 #++
+module BasicData
+  class BasicDataSeeder
 
-# add seeds here, that need to be available in all environments
+    def self.seed!
+      puts ' ↳ Builtin Roles'
+      BuiltinRolesSeeder.seed!
 
-%w{ builtin_roles
-    admin_user }.each do |file|
-  puts "**** #{file}"
+      puts ' ↳ Roles'
+      RoleSeeder.seed!
 
-  require "#{Rails.root}/db/seeds/#{file}"
+      puts ' ↳ Activities'
+      ActivitySeeder.seed!
```

Comment:
Remember my suggestion from above? If we did that (even if we didn't but let's ignore that for a moment) we could just give the BasicDataSeeder a list of Seeder instances and call them here instead of writing out every line.

```ruby
class BasicDataSeeder
  attr_reader :seeders

  def initialize(seeders = default_seeders)
    @seeders = seeders
  end

  def seed!
    seeders.select(&:applicable?).map(&:seed!)
  end

  def default_seeders
    [ColorSeeder, WorkflowSeeder, PrioritySeeder, ...].map(&:new)
  end
end
```